tests/manage/pv_services/pvc_resize/test_node_restart_during_pvc_expansion.py

```diff
@@ -86,10 +86,51 @@ def test_worker_node_restart_during_pvc_expansion(self, nodes):
         # Verify pvc expansion status
         for pvc_obj in self.pvcs:
             assert pvc_obj.expand_proc.result(), (
-                f"Expansion failed for PVC {pvc_obj.name}"
+                f"Expansion failed for PVC {pvc_obj.name}\nDescribe output "
+                f"of PVC and PV:\n{pvc_obj.describe()}\n"
+                f"{pvc_obj.backed_pv_obj.describe()}"
             )
         log.info("PVC expansion was successful on all PVCs")
 
+        log.info("Verifying new size on pods.")
+        for pod_obj in self.pods:
+            if pod_obj.pvc.volume_mode == 'Block':
+                log.info(
+                    f"Skipping check on pod {pod_obj.name} as volume "
+                    f"mode is Block."
+                )
+                continue
+
+            # Wait for 240 seconds to reflect the change on pod
+            log.info(f"Checking pod {pod_obj.name} to verify the change.")
+            for df_out in TimeoutSampler(
+                240, 3, pod_obj.exec_cmd_on_pod, command='df -kh'
+            ):
+                df_out = df_out.split()
+                new_size_mount = df_out[
+                    df_out.index(pod_obj.get_storage_path()) - 4
+                ]
+                if new_size_mount in [
+                    f'{pvc_size_expanded - 0.1}G',
+                    f'{float(pvc_size_expanded)}G',
+                    f'{pvc_size_expanded}G'
+                ]:
+                    log.info(
+                        f"Verified: Expanded size of PVC {pod_obj.pvc.name} "
+                        f"is reflected on pod {pod_obj.name}"
+                    )
+                    break
```

Comment:
Do you want to add assert if new_size_mount is not in the pvc_size-expanded list ?

Reply:
Retry will be done. If it not reflected within the specified time, TimeOutExpired error will be raised.